npm/translatePolicy.go

```diff
@@ -273,10 +278,21 @@ func translateIngress(ns string, targetSelector metav1.LabelSelector, rules []ne
 		}
 
 		// fromRuleExists
-		for _, fromRule := range rule.From {
+		for j, fromRule := range rule.From {
 			// Handle IPBlock field of NetworkPolicyPeer
 			if fromRule.IPBlock != nil {
 				if len(fromRule.IPBlock.CIDR) > 0 {
+					ipCidrs[i] = append(ipCidrs[i], fromRule.IPBlock.CIDR)
+					cidrIpsetName := name + "-cidr" + strconv.Itoa(i) + "in"
+					if len(fromRule.IPBlock.Except) > 0 {
+						for _, except := range fromRule.IPBlock.Except {
+							ipCidrs[i] = append(ipCidrs[i], except + " " + util.IpsetNomatch)
```

Comment:
we shouldn't have no matches since we're moving to allow based style

Reply:
We can do it in next PR, let's have this PR focus on ip table to ip set change.

Reply:
Maybe a TODO comment can help move this work to our next PR.

Reply:
Let me add TODO comment.